src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>Waneiac</title>
```

Comment:
I'm sorry but cats are not eligible to signing the CLA. Therefore, I cannot accept this PR.

Reply:
I can take over this PR to get it accepted if you want!

Reply:
I don't usually allow it but this time I'll _leggo_.